main.go

```diff
@@ -97,28 +98,40 @@ type config struct {
 	GenericWebhookURLs []string `yaml:"generic_webhook_urls"`
 }
 
-// RejectionCondition contains the fields that should match a bug report for it to be rejected.
+// RejectionCondition contains the fields that can match a bug report for it to be rejected.
 type RejectionCondition struct {
-	// Required field: if a payload does not match this app name, the condition does not match.
+	// All the optional fields must match for the rejection condition to apply
```

Comment:
> 	// All the optional fields must match for the rejection condition to apply

this is the wrong place for this comment: it looks like it's only relevant to `App` (and will be included as part of the generated documentation for `App`). Put it outside the struct please.